docs/how-to-guides/feathr-configuration-and-env.md

```diff
@@ -76,9 +76,10 @@ Feathr will get the configurations in the following order:
-| FEATURE_REGISTRY__PURVIEW__PURVIEW_NAME               | Configure the name of the purview endpoint.                                                                                                                                                                                                                                        | Required if using Purview as the endpoint.                                |
-| FEATURE_REGISTRY__PURVIEW__DELIMITER                  | See [here](#FEATURE_REGISTRY__PURVIEW__DELIMITER) for more details.                                                                                                                                                                                                                     | Required                                                                  |
-| FEATURE_REGISTRY__PURVIEW__TYPE_SYSTEM_INITIALIZATION | Controls whether the type system (think this as the "schema" for the registry) will be initialized or not. Usually this is only required to be set to `True` to initialize schema, and then you can set it to `False` to shorten the initialization time.                          | Required                                                                  |
+| FEATURE_REGISTRY__API_ENDPOINT                        | Specifies registry endpoint.                                                                                                                                                                                                                                                       | Required if using registry service.                                       |
+| FEATURE_REGISTRY__PURVIEW__PURVIEW_NAME               | [Deprecated Soon] Configure the name of the purview endpoint.                                                                                                                                                                                                                                        | Required if using Purview as the endpoint.                                |
+| FEATURE_REGISTRY__PURVIEW__DELIMITER                  | [Deprecated Soon] See [here](#FEATURE_REGISTRY__PURVIEW__DELIMITER) for more details.                                                                                                                                                                                                                     | Required                                                                  |
+| FEATURE_REGISTRY__PURVIEW__TYPE_SYSTEM_INITIALIZATION | [Deprecated Soon] Controls whether the type system (think this as the "schema" for the registry) will be initialized or not. Usually this is only required to be set to `True` to initialize schema, and then you can set it to `False` to shorten the initialization time.                          | Required                                                                  |
```

Comment:
Clarify the last 3 configs are `Required if user directly call Purview backend`. 
Could also add a section `Registry Service Update` in follow up PRs to explain the user experience difference and link to [Deprecated Soon].